docs/notebooks/demo_Cometary_Activity.ipynb

```diff
@@ -0,0 +1,1047 @@
+      "text/plain": [
+       "<Figure size 1000x800 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(figsize=(10, 8))\n",
+    "ax.plot(observations_df[\"Obj_Sun_LTC_km\"]/1.495978707e8 , observations_df[\"Simple_mag\"], linestyle=\"-\", label=\"No phase curve\", color='purple')\n",
+    "\n",
+    "ax.legend()\n",
+    "ax.set_ylabel(\"Apparent magnitude\")\n",
+    "ax.set_xlabel(\"Heliocentric distance (au)\")\n",
+    "plt.gca().invert_yaxis()\n",
+    "plt.grid()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "250e3f6f",
+   "metadata": {},
+   "source": [
+    "The effect of the cometary activity class is compute the apparent magnitude of the active object from an input apparent magnitude of the nucleus. The entire `observational_df` dataframe is exposed to the cometary activty class, so any dependencies can be added. \n",
+    "\n",
+    "Let's use the LSSTCometActivity class from `sorcha_addons`. We need the following columns in our dataframe:\n",
+    "\n",
+    "    * ``afrho1\"`` = V-band Afρ value of the comet at 1 au\n",
```

Comment:
Is this supposed to be formatted as a code block?  Regardless, the units on Afρ (cm in this case) should be stated here or somewhere near here.

Reply:
done